docs/docs/configuration/hardware_acceleration.md

```diff
@@ -15,6 +15,78 @@ ffmpeg:
   hwaccel_args: preset-rpi-64-h264
 ```
 
+### Orange Pi 5 ( ArmNN )
+
+Ensure you have installed
```

Comment:
Can we make this an install script similar to the way TensorRT detector has?

Reply:
I'm afraid we need to stop overwhelming Dockerfile with if/if/else spaghetti and have Dockerfile / board / arch

Reply:
There is no sense to have armnn related detector for x86_64 board. No need to install packages from rkmpp scope for raspberry PI board, no need to have rknpu libraries for raspberry pi
We should stop using raspberry pi PPAs for not raspberry PI boards

Reply:
As an architect I see us start decoupling frigate from Docker
frigate should become an app, which is used by specific Docker as part of the whole system, depending from the hardware available

Reply:
Got it, makes sense @NickM-27 
I'll check if I can add proper PPAs and script for installing proper ffmpeg for hardware acceleration  on this board